ui/package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint",
+    "lint": "npx eslint .",
```

Comment:
You probably still want to run `next lint` as well, as I think that includes some additional `next` specific things.

Reply:
Also this is a no-op in CI because the CI runs `npm test`, which runs `next lint` directly and not `npm run lint`.

Reply:
ugh!

Reply:
```suggestion
    "lint": "npx eslint . && next lint",
```